app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    items_dic = {}
+
+    def wrapper(*args) -> int:
```

Comment:
Indicate that `wrapper` could return the `Any` type in the return type annotation. The function result potentially is not limited to only integers.